exchange/bidder.go

```diff
@@ -54,7 +55,7 @@ type AdaptedBidder interface {
 	//
 	// Any errors will be user-facing in the API.
 	// Error messages should help publishers understand what might account for "bad" bids.
-	requestBid(ctx context.Context, bidderRequest BidderRequest, conversions currency.Conversions, reqInfo *adapters.ExtraRequestInfo, adsCertSigner adscert.Signer, bidRequestOptions bidRequestOptions, alternateBidderCodes openrtb_ext.ExtAlternateBidderCodes) ([]*pbsOrtbSeatBid, []error)
+	requestBid(ctx context.Context, bidderRequest BidderRequest, conversions currency.Conversions, reqInfo *adapters.ExtraRequestInfo, adsCertSigner adscert.Signer, bidRequestOptions bidRequestOptions, alternateBidderCodes openrtb_ext.ExtAlternateBidderCodes, hookExecutor hookexecution.StageExecutor) ([]*pbsOrtbSeatBid, []error)
```

Comment:
This change is probably a merge issue, for AuctionResponseStage `hookExecutor hookexecution.StageExecutor` is not needed here (it's needed for other stages). May you please check this, looks like this PR can be smaller. Unless I miss something.

Reply:
Yes, you are correct. I guess I moved it by cherry-picking together with some other useful changes. But the hookExecutor is not needed there.